### PR TITLE
Add branch_alias.yml for all packages

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

**Review by jderusse — APPROVED** [review]
Thank you﻿

**Comment by Nyholm** [comment]
Thank you

File: src/Integration/Aws/DynamoDbSession/.github/workflows/branch_alias.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      - name: Set up PHP
+        uses: shivammathur/setup-php@2.5.0
+        with:
+          php-version: 7.4
+          coverage: none
+
+      - name: Find branch alias
+        id: find_alias
+        run: |
+          TAG=$(echo $GITHUB_REF | cut -d'/' -f 3)
```
Comment (bendavies) [inline src/Integration/Aws/DynamoDbSession/.github/workflows/branch_alias.yml:22]: this doesn't allow for a tag with a `/` in it, (unlikely, i know)
we use 
`${GITHUB_REF#refs/tags/}`

Comment (Nyholm) [reply src/Integration/Aws/DynamoDbSession/.github/workflows/branch_alias.yml:22]: Cool. I didnt know of that syntax. 

Feel free to send a PR to update this. I usually copy the actions from this repo time to time, so updating might benefit other repos. =)

```
~
❯ GITHUB_REF="refs/tags/0.1.1"

~
❯ echo $GITHUB_REF
refs/tags/0.1.1

~
❯  echo $GITHUB_REF | cut -d'/' -f3
0.1.1

~
❯ echo ${GITHUB_REF#refs/tags/}
0.1.1
```

**Pull request merged**